### ISSUE TITLE
add benchmark scene

### ISSUE THREAD

**Issue by roboman2444** [yvt/openspades#513]
Would be really nice to have a standard benchmark scene so i don't have to guess if my small modifications are actually improving performance or not.

**Comment by Tapmemer** [comment]
you mean like a map where a spectator enttity (or cam entity) flys around

**Comment by roboman2444** [comment]
Yes, but also with some player entities running around and doing things.
#517 Would be useful for this.
